lib/runners/clojure.js

```diff
@@ -1,18 +1,20 @@
-var shovel = require('../shovel');
+var shovel = require('../shovel'),
+    path = require('path');
 
 module.exports.run = function run(opts, cb) {
+    var uberJar = path.resolve(__dirname, '../../jvm-runner/target/jvm-runner-0.1.1-standalone.jar');
```

Comment:
I think groovy started failing by this commit see #175  I don't think it's by not using carmine but perhaps it's due to it's jvm-runner version [jvm-runner-0.1.2-standalone.jar](https://github.com/Codewars/codewars-runner-cli/blob/master/lib/runners/groovy.js#L5) what do you think mr @xcthulhu ? :disappointed: